src/FirstSteps/RPG/Heroes/Hero.cs

```diff
@@ -37,81 +38,7 @@ public void DisplayStats()
             .AddItem("Agility", _agility, Color.Yellow)
             .AddItem("Health", _health, Color.Red)
             .AddItem("Damage", _damage, Color.Green));
-            Console.WriteLine($"Equipment:\ntotal price: {DisplayTotalPrice()}\ntotal items: {DisplayTotalItems()}\ntotal weight; {DisplayTotalWeight()} ");
-        }
-
-        public bool AddItemToBackpack(Item item)
-        {
-            if (!CanHandleSpecialItem(item))
-            {
-                return false;
-            }
-            else if (backPack.Count <= maxBackpack && CountWeight() <= _maxWeight && item.Weight + CountWeight() <= _maxWeight)
-            {
-                backPack.Add(item);
-                return true;
-            }
-            else
-            {
-                return false;
-            }
-        }
-        protected abstract bool CanHandleSpecialItem(Item item);
```

Comment:
`CanHandleSpecialItem` should be still a hero's method. You know, Dwarf will decide if he is able to handle Axe or not. It's not an equipment responsibility.